SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -343,11 +383,22 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Runtime", AdvancedDisplay)
 	FRuntimeVariantVersion CompatibilityModeRuntimeVersion;
 
+	/** Returns the version information for the currently set inspector*/
+	const FInspectorVersion& GetSelectedInspectorVersion() const
```

Comment:
could this be simplified to simply return an FString? I'm not sure we need the `FInspectorVersion` struct cause it only stores a string and a bool and we would only ever want to return the Inspector version based on what is pinned and whether the bool is true or false.
or will this be extended later to store more information?

Reply:
Right now we need only the pinned version, but in the future we're expecting to have different Inspector versions and to be able to support all of them, so that's why we thought this could be useful

Reply:
even then we would only need an FString to specify the version, right? I would say let's keep it simple until we know exactly what extra information we would need to store :)